engines/kyra/detection_tables.h

```diff
@@ -1500,6 +1500,23 @@ const KYRAGameDescription adGameDescs[] = {
 		LOL_FLOPPY_FLAGS
 	},
 
+	{ // Japanese PC9821 version - Bug 9828
+		{
+			"lol",
+			"Extracted",
+			{
+				{ "GENERAL.PAK", 0, "8902b233405461025e25093b9c4c8d2c", -1 },
+				{ "CHAPTER7.PAK", 0, "32a46c338a3d6249c273cc3b3881bb8a", -1 },
+				{ 0, 0, 0, 0 }
+			},
+			Common::JA_JPN,
+			Common::kPlatformDOS,
```

Comment:
This should probably be Common::kPlatformPC98, not Common::kPlatformDOS

Reply:
Good catch, fixed in a2c31c1